adala/skills/_base.py

```diff
@@ -44,6 +45,8 @@ class Skill(BaseModelInRegistry):
         description="Instructs agent what to do with the input data. "
         "Can use templating to refer to input fields.",
         examples=["Label the input text with the following labels: {labels}"],
+        # TODO: instructions can be deprecated in favor of using `input_template` to specify the instructions
```

Comment:
is there a ticket to capture the various things that should be deprecated? instructions, output_template+field_schema, get_llm_response, others?

Reply:
also, once output_template+field_schema are gone does that mean the pydantic_generator utils will not be needed? Or is there another use of json schemas?

Reply:
I don't have a strong opinion on that, I want to start working on unified approach of defining a structured prediction then we will see what interface can be the best for that, and dropping the rest.